src/App.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Quotes from './components/Quotes';
+import About from './components/About';
 import './App.css';
 
-const App = () => (
-  <div className="app">
-    <Calculator />
-  </div>
-);
-
-export default App;
+export default function App() {
+  return (
+    <Router>
+      <Navbar />
+      <Switch>
+        <Route path="/calculator">
+          <Calculator />
+        </Route>
```

Comment:
:heavy_check_mark: Excellent job. Everything looks okay. :+1: 

[OPTIONAL]

- [ ] It'll be nice to add `LICENCE` to your README file.

Reply:
Added thanks!